packages/builder/src/builtin-tasks/task-names.ts

```diff
@@ -4,10 +4,12 @@ export const TASK_HELP = "help";
 export const TASK_CONSOLE = "console";
 export const TASK_CLEAN = "clean";
 export const TASK_INIT = "init";
-
+export const TASK_NODE_INFO = "node-info";
```

Comment:
Can we simply have `info`? It's a long task name

Reply:
What would `info` mean? Here I want to print the connection and node information.